api/src/main/java/io/opentelemetry/api/trace/SpanId.java

```diff
@@ -93,4 +93,9 @@ public static boolean isValid(String spanId) {
   public static String bytesToHex(byte[] spanId) {
     return BigendianEncoding.toLowerBase16(spanId);
   }
+
+  /** Convert the the given hex spanId into a long representation. */
+  public static long asLong(String src) {
```

Comment:
Can you add some small unit tests for these trace utility change?

Reply:
yes, good idea. I will do so.

Reply:
done

Reply:
We should consider to move this class in an api extension, I don't think it should be part of the public API .

Reply:
I would like to do that, but I'd rather not make a verbatim copy of the BigendianEncoding class, since there are a bunch of fiddly bits in there that it would be better not to have duplicates of. Any thoughts on that?

Reply:
@bogdandrutu any ideas on how to tackle this? Should we follow-up with a separate issue/PR?